src/App.js

```diff
@@ -1,3 +1,40 @@
+import {useState} from 'react';
+import styled from 'styled-components';
+
+import TodoForm from './components/TodoForm.js';
+
 export default function App() {
-  return <div>App goes here</div>;
+  const [todos, setTodos] = useState([]);
+
+  return (
+    <Grid>
+      <Scroller>
+        {todos.map((todo, index) => (
+          <TodoItem key={index}>{todo}</TodoItem>
+        ))}
```

Comment:
I am not sure about using index as a key. This might be dangerous in the future.

Reply:
I know, but I am sure this won't be a problem. I will introduce ids later.